src/email-templates/approval_notification.handlebars

```diff
@@ -1,9 +1,9 @@
-<h1>Glad to have you on board, {{companyName}}!</h1>
-<p>We decided to accept your application, and you are just one step away from being able to use NIJobs!</p>    
-<p>Now, you can log in to <a href="https://ni.fe.up.pt/nijobs">NIJobs</a> and complete your registration.
-Once you finish, you'll be able to place some work offers!</p>
+<h1>We are delighted to have you on board, {{companyName}}!</h1>
+<p>We are pleased to inform you that your application has been accepted, and all the offers
+    you have created so far are now visible to the public!</p>
+<p>Going forward, any offer you create will be immediately visible to the public audience.</p>
```

Comment:
We are pleased to inform you that your application has been accepted, and all the offers you have created are now accessible to the public!
From now on, any offer you create will be immediately available to the public audience.